src/pages/login.page.tsx

```diff
@@ -0,0 +1,95 @@
+import { css } from '@emotion/react';
+
+import LogoBig from '~/assets/LogoBig';
+import KakaoLoginButton from '~/components/kakaoLoginButton/KakaoLoginButton';
+import colors from '~/styles/color';
+
+export default function Login() {
+  return (
+    <div css={LoginPageWrapper}>
+      <section className="page-top">
+        <main className="title">나의 커리어 DNA 연구</main>
```

Comment:
타이틀부분을 `h1` 으로 묶어주는게 어떨까요?? 

```suggestion
        <main><h1 css={titleCss}>나의 커리어 DNA 연구</h1></main>
```

Reply:
옙!